generate.py

```diff
@@ -17,10 +17,11 @@
 
 @torch.no_grad()
 def generate(
```

Comment:
The other scripts that call this function also need an update

Reply:
I'll try to get to this later today

Reply:
I fixed adapter. Need to do a check across the repo to see what's outdated.

Can you update the inference numbers in the tpu howto?

Reply:
Waiting to update these numbers for https://github.com/Lightning-AI/lit-parrot/pull/83#issuecomment-1558042678